src/hera/workflows/script.py

```diff
@@ -31,6 +36,21 @@
 from hera.workflows.task import Task
 
 
+class ScriptConstructor(BaseMixin):
```

Comment:
Can this have some docs on what users need to implement primarily and what independent transforms can perform? Just a bit to provide some clarity